examples/gno.land/r/chaegnal/organizer/group/group_actions.gno

```diff
@@ -0,0 +1,39 @@
+package group
+
+import (
+	"std"
+
+	"gno.land/p/demo/ownable"
+
+	"gno.land/r/chaegnal/organizer"
+)
+
+// DeleteFriendGroup allows the owner to delete the friendgroup
+// Panics if anyone but the owner try to delete the group
+func DeleteFriendGroup(cur realm, groupId, logged string) {
+	group := organizer.GetGroupFromId(groupId)
+
+	if group.Owner.Owner() != std.Address(logged) {
+		panic("only the owner can delete the group")
+	}
```

Comment:
Recommendation for security, you should never trust input values, in this case using `std.PreviousRealm().Address()` is one way to make sure group owner address can be faked:
```suggestion
func DeleteFriendGroup(cur realm, groupId string) {
	logged := std.PreviousRealm().Address()
	group := organizer.GetGroupFromId(groupId)

	if group.Owner.Owner() != logged {
		panic("only the owner can delete the group")
	}
```